### ISSUE TITLE
[Next] Nested Routes are not created with pages options

### ISSUE THREAD

**Issue by Applelo** [nuxt-modules/i18n#1649]
### Version
@nuxtjs/i18n: 8.0.0-beta.4
nuxt: 3.0.0

### @nuxtjs/i18n configuration
```
i18n: {
    defaultLocale: 'fr_FR',
    locales: [{ code: 'fr_FR', iso: 'fr-FR', file: 'fr.yml' }],
    langDir: './locales/',
    parsePages: false,
    pages: {
      account: {
        fr_FR: '/compte',
      },
      //not working
      'account/profile': {
        fr_FR: '/compte/profil',
      },
      'account/addresses': {
        fr_FR: '/compte/adresses',
      },
    },
  }
```

### Reproduction Link
[Reproduction on StackBlitz](https://stackblitz.com/edit/nuxt-i18n-nested-routes?file=nuxt.config.ts)

### Steps to reproduce

Try to access to the translated route for profile page, it is not working and return a 404 error.

### What is Expected?

Nested root should work (like https://v3.nuxtjs.org/guide/directory-structure/pages/#nested-routes) with `pages` configuration. I tried with `defineI18nRoute` and no problem.

### What is actually happening?

Nested routes are not created or linked I guess. Maybe I miss something.

**Comment by kazupon** [comment]
Thank you for your reporting!
and, sorry late my reply.

I'll try to fix this issue.

**Comment by kazupon** [comment]
I've just fixed this issue and released out edge version  `@nuxtjs/i18n-edge@8.0.0-beta.4-27824203.a7fc4de`
As you can see in #1673, I have rewritten custom routes.
So you need to read the docs.
https://v8.i18n.nuxtjs.org/guide/custom-paths

I will close this issue with the release of v8.0.0-beta.5.
Thanks!
